src/wp-includes/class-wp-autoloader.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Class WP_Autoloader
+ *
+ * WordPress Class, Interface and Trait Autoloader
+ *
+ */
+class WP_Autoloader {
+
+	/**
+	 * @var array
+	 */
+	private static $_classmap;
+
+	/**
+	 * WP_Autoloader constructor.
+	 */
+	function __construct() {
+
+		if ( ! defined( 'DISABLE_CORE_AUTOLOADER' ) ) {
```

Comment:
Please look again.  The state of the code when you commented use the following `if()` statement:

    if ( defined( 'USE_COMPOSER_AUTOLOADER' ) && USE_COMPOSER_AUTOLOADER  ) {